tools/runners/run-log.js

```diff
@@ -105,6 +105,19 @@ Object.assign(RunLog.prototype, {
     }
   },
 
+  rawLog: function (msg) {
+    var self = this;
+    var obj = {
+      time: new Date,
+      message: msg
+    };
+    self._record(obj);
```

Comment:
It does not seem to make sense to store this log message if we're doing raw output here. Read the purpose of this module in the comment above (it will collect app logs, and show them in the browser, etc).

I guess this actually implies that if really want a raw output, we might as well do `console.info()` (or Console.rawInfo?) and not bother (i.e. we didn't need to touch this module at all).

Reply:
We need to clear the special, but removed the recording.